website/src/jest/css/jest.css

```diff
@@ -920,6 +920,15 @@ div[data-twttr-id] iframe {
   opacity: 1;
 }
 
+/** Blog Images **/
+img[alt="perf-basic-scheduling"], img[alt="perf-improved-scheduling"] {
```

Comment:
I'll update this for future blog posts. I don't think I want all the images have the same size. Unfortunately our version of markdown in `Marked.js` doesn't support specifying the width/height and `<img />` tags don't work for some reason :) this was a quick hack, I guess :)